src/silo/storage/sequence_store.cpp

```diff
@@ -1,19 +1,27 @@
 #include "silo/storage/sequence_store.h"
 
+#include <spdlog/spdlog.h>
 #include <tbb/blocked_range.h>
 #include <tbb/enumerable_thread_specific.h>
 #include <tbb/parallel_for.h>
-#include <tbb/parallel_for_each.h>
 #include <atomic>
-#include <iostream>
 #include <roaring/roaring.hh>
 #include <string>
-#include <syncstream>
 #include <vector>
 
-#include "external/PerfEvent.hpp"
+#include "silo/common/format_number.h"
 #include "silo/common/silo_symbols.h"
-#include "silo/database.h"
+
+auto fmt::formatter<silo::SequenceStoreInfo>::format(
+   silo::SequenceStoreInfo sequence_store_info,
+   fmt::format_context& ctx
+) -> decltype(ctx.out()) {
+   return format_to(
+      ctx.out(), "SequenceStoreInfo[sequence count: {}, size: {}, N bitmaps size: {}]",
+      sequence_store_info.sequence_count, sequence_store_info.size,
```

Comment:
Maybe I'll have a look at the formatter. In my eyes, this should be on separate lines :)

Reply:
Yes, that would suit my eyes better, too ;) There are a couple of weird line breaks in the code now.